### DISCUSSION TITLE
Proposta d'esame Matteo Migliorati

### DISCUSSION THREAD

**Discussion by Matteogit05** [deneb-alpha/Lab_open_source-2024_2025#23] [Projects]
Richieste: 
- Cos'è una licenza
- A cosa servono e perché sono importanti
- Illustrazione delle licenze più utilizzate
- Come trovare la licenza adatta
- Esempi

Opzionale:
- Panoramica su Openstreetmap e la licenza che utilizza
- Come funziona la community OSM e come contribuire

**Comment by deneb-alpha** [comment]
Ciao @Matteogit05,
la proposta mi sembra completa.
